src/sentry/static/sentry/app/views/projectPluginDetails2.jsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
```

Comment:
did you mean for this to be in the diff?

Reply:
yes